default/main.py

```diff
@@ -21,11 +26,30 @@ async def index(request: Request):
 
 
 @app.post('/userdata/')
-async def userdata(request: Request, apikey: str = Form(...), webhook: str = Form(...)):
+async def userdata(request: Request, apikey: str = Form(...), webhook: str = Form(...), vt_query: str = Form(...)):
     re_url = re.compile(r'[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)')
     if not re_url.search(webhook):
         raise HTTPException(400, 'Bad request')
     with client.context():
-        task = models.Userdata(apikey=apikey, webhook=webhook)
+        task = models.Userdata(apikey=apikey, webhook=webhook, vt_query=vt_query)
         task.put()
     return templates.TemplateResponse("index.html", {'request': request, 'content': 'Your answer was successfully saved'})
+
+
+@app.get('/run_queries/')
+async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
+    if x_appengine_cron != 'true':
+        raise HTTPException(403, 'Access forbidden')
+
+    with client.context():
+        for user in models.Userdata.query():
+            async with aiohttp.ClientSession() as httpSession:
+                async with httpSession.get(
+                        'https://www.virustotal.com/api/v3/intelligence/search?query={query}'.format(query=user.vt_query),
+                        headers={'x-apikey': user.apikey},
+                        ssl=ssl_context
+                    ) as resp:
+                    json = await resp.json()
+                    assert resp.status == 200
```

Comment:
Never use `assert` in production code. Asserts are omitted when optimization is requested (https://docs.python.org/3/reference/simple_stmts.html#assert):
> The current code generator emits no code for an assert statement when optimization is requested at compile time.